logprep/runner.py

```diff
@@ -151,6 +151,8 @@ def start(self):
         self._logger.info("Shutting down")
         self._manager.stop()
         self._logger.info("Shutdown complete")
+        if self._manager.loghandler is not None:
+            self._manager.loghandler.stop()
```

Comment:
```suggestion
        if self._manager.loghandler is not None:
            self._manager.loghandler.stop()
        self._logger.info("Shutdown complete")
```
I hope we can still use the regular logger after the loghandler was stopped.

Reply:
no ... that's the reason, this won't work. After stopping the handler, no logs will be handled bei the console handler